docs/src/bot.md

```diff
@@ -143,13 +103,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        # Uncomment other versions if you want multi-version signatures (should exactly match BotConfig.version):
-        version:
-          - '1.4.2'
+        version:   # NOTE: the versions below should match those in your botconfig
+          - '1.4.1'
```

Comment:
The version specified in the docs was 1.4.1, so I changed it here.

But, do we have to pass them to `BotConfig`? Could it just extract versions from what's run in the workflow file? Where possible I prefer to have only one place to set each configuration, having two makes things more fragile when they are not synchronized (as already happened here).

Reply:
The reason for this is because the workflow file (written in TOML) is independent of the `BotConfig` (written in Julia). `BotConfig` option is used to generate the `precompile_includer` file, and these options here in TOML, are the actual versions that are used.

However, with that being said, We can write a Julia code to extract these versions from the TOML file. It should be pretty easy using `Pkg.TOML`. 🤔